daemon/cluster/services.go

```diff
@@ -50,14 +50,16 @@ func (c *Cluster) GetServices(options apitypes.ServiceListOptions) ([]types.Serv
 		return nil, err
 	}
 
+	if len(options.Filters.Get("runtime")) == 0 {
+		// Default to using the container runtime filter
```

Comment:
Wondering if we should do this in the CLI instead of the API. No strong preference though.